features/admin.branding.v1/components/custom-text/custom-text-fields.tsx

```diff
@@ -155,6 +163,94 @@ const CustomTextFields: FunctionComponent<CustomTextFieldsProps> = (props: Custo
         );
     };
 
+    const handleMultiAttributeLink = (): void => {
+        if (serverConfigurationConfig.dynamicConnectors === true) {
+            history.push(
+                AppConstants.getPaths().get(
+                    "GOVERNANCE_CONNECTOR_EDIT"
+                ).replace(
+                    ":categoryId",
+                    ServerConfigurationsConstants.
+                        ACCOUNT_MANAGEMENT_CONNECTOR_CATEGORY_ID
+                ).replace(
+                    ":connectorId",
+                    ServerConfigurationsConstants.
+                        MULTI_ATTRIBUTE_LOGIN_CONNECTOR_ID
+                )
+            );
+        } else {
+            history.push(AppConstants.getPaths().get("ALTERNATIVE_LOGIN_IDENTIFIER_EDIT"));
+        }
+    };
+
+    const getMultiAttributeLabel = (): string => {
+        if (serverConfigurationConfig.dynamicConnectors === true) {
```

Comment:
Let's not rely on this config. Cos in the future if we decide to refactor the governance connector logic this would get unnecessarily affected.